k8s/argocd/local/app-of-apps.values.yaml

```diff
@@ -1,2 +1,5 @@
 chartVersions:
+  wbaasUi: 0.4.0
```

Comment:
I love being explict here; better than the implicit values from the chart

Reply:
also allows deployment without a new chart